packages/replay/src/eventBuffer/EventBufferArray.ts

```diff
@@ -18,6 +18,11 @@ export class EventBufferArray implements EventBuffer {
     return this.events.length > 0;
   }
 
+  /** @inheritdoc */
+  public get type(): EventBufferType {
+    return 'default';
```

Comment:
Hmm, not sure about `default`, as actually it is not the default 😅 what about `sync` or something like that, maybe?